services/transaction_receipt.proto

```diff
@@ -164,4 +164,9 @@ message TransactionReceipt {
      * the newly created NFTs
      */
     repeated int64 serialNumbers = 14;
+
+    /**
+     * In the receipt of a NetworkCreateNode, the id of the newly created node.
```

Comment:
`NetworkCreateNode` is not actually the name of the transaction anymore

Reply:
Updated.